docs/reference/db-config.rst

```diff
@@ -1,8 +1,28 @@
 :orphan:
 
-=======================
-db/config.cc properties
-=======================
+======================
+db/config.cc reference
+======================
 
-.. datatemplate:yaml:: ../_data/db_config.yaml
+Configuration properties
+------------------------
+
+.. dbconfigdatatemplate:: ../_data/db_config.yaml
   :template: db_config.tmpl
+  :filter: Used
+
+
+Deprecated properties
+---------------------
+
+As ScyllaDB evolves, certain configuration properties may become obsolete or redundant.
+In our commitment to ensure smooth transitions, we have opted to keep this properties documented.
```

Comment:
this -> these

Reply:
Resolving, it refers to a previous patch.